tools/gmon/README.md

```diff
@@ -0,0 +1,141 @@
+# Cloud Monitoring CLI
+
+`gmon` is a command-line interface for Cloud Monitoring written in Python.
+
+The CLI supports the following endpoints:
+
+-   `metrics` _(GA)_: Manage Cloud Monitoring metrics.
+-   `accounts` _(EAP)_: Manage Cloud Monitoring accounts.
+-   `services` _(ALPHA)_: Manage Cloud Monitoring services.
+-   `slos` _(ALPHA)_: Manage Cloud Monitoring service level objectives.
+
+Other endpoints are available directly in `gcloud alpha monitoring`, and won't
+be added to `gmon`, in particular:
+
+-   `channel-descriptors`: _(ALPHA)_ Cloud Monitoring notification channel descriptors.
+-   `channels`: _(ALPHA)_ Manage Cloud Monitoring notification channels.
+-   `policies` _(ALPHA)_: Manage Cloud Monitoring alerting policies.
+-   `dashboards` _(GA)_: Manage Cloud Monitoring dashboards.
+
+The goal of this CLI is to fill the gap and simplify users life to query
+monitoring endpoints in a more friendly manner and troubleshoot their monitoring
+setups.
+
+### Metrics
+
+The `gmon metrics` command allow one to interact with Google Cloud Monitoring metrics.
+
+You can get more information on the command by running `gmon metrics --help`, but
+here is a cheatsheet:
+
+    # List all metrics in project
+    gmon metrics list -p <PROJECT_ID>
```

Comment:
bit `gmon` invocations should be formatted w/ backticks

Reply:
done